lua/framework/logging/lua_log.cc

```diff
@@ -3,17 +3,17 @@
 #include "framework/runtime.h"
 #include "framework/logging/log.h"
 
-#include "log_lua.h"
+#include "lua_log.h"
 #include "framework/console/console.h"
 
 using namespace opensn;
 
 namespace opensnlua
 {
-RegisterLuaFunctionAsIs(LogSetVerbosity);
-RegisterLuaFunctionAsIs(Log);
-RegisterLuaFunctionAsIs(LogProcessEvent);
-RegisterLuaFunctionAsIs(LogPrintTimingGraph);
+RegisterLuaFunctionNamespace(LogSetVerbosity, log, SetVerbosity);
+RegisterLuaFunctionNamespace(LogLog, log, Log);
```

Comment:
What is LogLog?

Reply:
The original function name was `Log`. I gave it a `Log` prefix, so it is consistent with `LogProcessEvent`, `LogSetVerbosity`, etc. Users just see `Log`...

Reply:
If all of these routines are within the `log` namespace, should we have the `Log` prefix?